frontend/src/scenes/insights/aggregationAxisFormat.ts

```diff
@@ -0,0 +1,38 @@
+import { LemonSelectOptions } from 'lib/components/LemonSelect'
+import { compactNumber, humanFriendlyDuration, percentage } from 'lib/utils'
+import { ChartDisplayType } from '~/types'
+
+const formats = ['numeric', 'duration', 'percentage'] as const
+export type AggregationAxisFormat = typeof formats[number]
+
+export const isAggregationAxisFormat = (candidate: unknown): candidate is AggregationAxisFormat =>
+    formats.includes(candidate as AggregationAxisFormat)
+
+export const aggregationAxisFormatSelectOptions = formats.reduce((target, format) => {
+    target[format as string] = { label: format as string }
```

Comment:
sorry one request - can we capitalize this to match all the other selects

Reply:
Ugh, what?! No! *deletes PR